packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -115,6 +116,356 @@ export class ApplicationLoadBalancer extends BaseLoadBalancer implements IApplic
       securityGroupId: this.securityGroup.export().securityGroupId,
     };
   }
+
+  /**
+   * Return the given named metric for this Application Load Balancer
+   *
+   * @default Average over 5 minutes
+   */
+  public metric(metricName: string, props?: cloudwatch.MetricCustomization): cloudwatch.Metric {
+    return new cloudwatch.Metric({
+      namespace: 'AWS/ApplicationELB',
+      metricName,
+      dimensions: { LoadBalancer: this.fullName },
+      ...props
+    });
+  }
+
+  /**
+   * The total number of concurrent TCP connections active from clients to the load balancer and from the load balancer to targets.
```

Comment:
wrap (I use [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) for VSCode, and it's awesomer than life)